### PR TITLE
EOS-26573:S3: Handle consul endpoint and port values in config phase

### PR DESCRIPTION
**Pull request by sachitanands**
Signed-off-by: Sachitanand Shelake <sachitanand.shelake@seagate.com>

# Problem Statement
- Problem statement

# Design
-  For Bug, Describe the fix here.
-  For Feature, Post the link for design

# Coding
   Checklist for Author
-  [ ] Coding conventions are followed and code is consistent

# Testing 
  Checklist for Author
- [ ] Unit and System Tests are added
- [ ] Test Cases cover Happy Path, Non-Happy Path and Scalability
- [ ] Testing was performed with RPM

# Impact Analysis
  Checklist for Author/Reviewer/GateKeeper
- [ ] Interface change (if any) are documented
- [ ] Side effects on other features (deployment/upgrade)
- [ ] Dependencies on other component(s)

# Review Checklist 
  Checklist for Author
- [ ] JIRA number/GitHub Issue added to PR
- [ ] PR is self reviewed
- [ ] Jira and state/status is updated and JIRA is updated with PR link
- [ ] Check if the description is clear and explained

# Documentation
  Checklist for Author
- [ ] Changes done to WIKI / Confluence page / Quick Start Guide

### CODE REVIEW

**Comment by sachitanands** [comment]
this PR has dependency on https://github.com/Seagate/cortx-prvsnr/pull/6290 
once above PR gets merged , we can merge our s3 PR .

File: scripts/provisioning/configcmd.py [hunk 1]
```diff
@@ -244,11 +244,9 @@ def push_s3_ldap_schema(self):
       ldap_lock = False
       self.logger.info('checking for concurrent execution scenario for s3 ldap scheam push using consul kv lock.')
       openldap_key=self.get_confkey("S3_CONSUL_OPENLDAP_KEY")
-      # TODO : update protocol and port
-      consul_endpoint_url=self.get_endpoint("CONFIG>CONFSTORE_CONSUL_ENDPOINTs", "fqdn", "tcp")
-      consul_endpoint_port=8500
+      consul_endpoint_url=self.get_endpoint("CONFIG>CONFSTORE_CONSUL_ENDPOINTS", "fqdn", "http")
```
Comment (nileshgovande) [inline scripts/provisioning/configcmd.py:247]: Use a default value here and we merge the same

Comment (sachitanands) [reply scripts/provisioning/configcmd.py:247]: done

**Review by nileshgovande — APPROVED** [review]

File: scripts/provisioning/configcmd.py (hunk 1 above)
Comment (sachitanands) [reply scripts/provisioning/configcmd.py]: done

**Comment by sachitanands** [comment]
triggered custom cortx-all image job : http://eos-jenkins.colo.seagate.com/job/GitHub-custom-ci-builds/job/centos-7.9/job/cortx-all-image-custom-ci/1275/

**Comment by sachitanands** [comment]
triggered new cortx-all image for testing : http://eos-jenkins.colo.seagate.com/job/GitHub-custom-ci-builds/job/centos-7.9/job/cortx-all-image-custom-ci/1292/

**Comment by sachitanands** [comment]
Tested changes with custom-build, and changes looks good.

image

**Pull request merged**